app/controllers/contacts_controller.rb

```diff
@@ -10,21 +11,25 @@ def index
 
   # GET /contacts/1 or /contacts/1.json
   def show
+    authorize @contact
   end
 
   # GET /contacts/new
   def new
+    authorize @contact
     @contact = Contact.new
```

Comment:
The `authorize` should happen below the creation of `@contact`, otherwise Pundit will be authorizing `nil`

```suggestion
    @contact = Contact.new
    authorize @contact
```

How does that sound?